appveyor.yml

```diff
@@ -149,14 +151,23 @@ before_build:
 - echo %PATH%
 - if "%platform%"=="mingw-w64" set PATH=%PATH:C:\Program Files\Git\usr\bin;=%
```

Comment:
This is the syntax for Windows batch file string replacements. The line replaces `C:\Program Files\Git\usr\bin` with the empty string in `%PATH%`.